app/src/main/java/org/listenbrainz/android/presentation/features/brainzplayer/ui/BrainzPlayerBackDropScreen.kt

```diff
@@ -90,9 +89,16 @@ fun BrainzPlayerBackDropScreen(
                 listenLiked = listenLiked,
                 isShuffled = isShuffled,
                 repeatMode = repeatMode,
-                modifier = Modifier.padding(paddingValues).alpha(1 - (delta / maxDelta).coerceIn(0f, 1f))
+                modifier = Modifier
+                    .padding(paddingValues)
+                    .alpha(1 - (delta / maxDelta).coerceIn(0f, 1f))     // FIXME : fps drops
+            )
+            SongViewPager(
```

Comment:
I don't remember what @yellowHatpro's take was on this. I remember him asking to revert these changes.

Reply:
Its still the same thing just a different way of writing. Will revert back.